srcStatic/Log.h

```diff
@@ -7,4 +7,19 @@ class Logger;
 
 
 void SetLogger(Logger* logger);
+void SetLoggerError(Logger* newLogger);
```

Comment:
Could we add a comment explaining what the difference between a Logger and LoggerError is. Based on reading what the issues we have on GitHub, I know what you are looking for, but just seeing this line of code I think would confuse me. The function names are not enough to distinguish to me.

Also, would recommend standardizing use of argument name to match between the two functions (whether the new prefix is kept or not).

Reply:
Ahh, thank you for noticing that. Seems I only updated it in the .cpp file.